src/main/resources/modules/adverse_childhood_experiences_module.json

```diff
@@ -0,0 +1,353 @@
+{
+  "name": "Adverse_Childhood_Experiences_Module",
```

Comment:
Have we reviewed this module as a team?

Reply:
This is a CODI leftover that should go into a separate PR. Same with the other modules. Apologies.